odbc/src/api/functions.rs

```diff
@@ -853,11 +853,6 @@ pub unsafe extern "C" fn SQLDisconnect(connection_handle: HDbc) -> SqlReturn {
             // set the mongo_connection to None. This will cause the previous mongo_connection
             // to drop and disconnect.
             *conn.mongo_connection.write().unwrap() = None;
-            // Temporary workaround for https://jira.mongodb.org/browse/RUST-1099
-            // This allows time for the underlying async runtime to clean up all of its
-            // resources before we report back success and subsequently drop the connection
-            // handle entirely
-            std::thread::sleep(std::time::Duration::from_millis(200));
```

Comment:
One small change request. It appears the driver team introduced the [shutdown](https://docs.rs/mongodb/2.7.1/mongodb/sync/struct.Client.html#method.shutdown) method in this [PR](https://github.com/mongodb/mongo-rust-driver/pull/920), and I think it would be best if we called that to ensure things are properly shut down.

Reply:
Done